PublicMapping/districtbuilder
# Project demographics export

## Overview

Adds demographic information support to the backend GeoJSON project districts export.

### Demo

![image](https://user-images.githubusercontent.com/4432106/84044232-4a3cfc80-a975-11ea-907a-0096c560b47b.png)

### Notes

 - Originally I had been planning on calculating these client-side as part of #97, but the district definition doesn't have enough information in it to query the static data ArrayBuffers on page load (we don't have the feature's ID / index). We do have this in the vector tile layer, so we can update demographic data dynamically without needing to query the server as the user interacts with the map.
 - Snuck a few unrelated fixes in here https://github.com/PublicMapping/districtbuilder/commit/fc6902239243159e49063a1073b82c3d255cab54, https://github.com/PublicMapping/districtbuilder/commit/041772e9141425d3659c9bb4f678f8a4d01a7609


## Testing Instructions

- `./scripts/server`
- Query the projects GeoJSON export endpoint. You should see demographic information included in the output.

Closes #160
